### ISSUE TITLE
InventoryOpener issue

### ISSUE THREAD

**Issue by ImDxni** [MinusKube/SmartInvs#187]
Hi, i was trying to do stuffs with SmartInvs, i was using the Dropper InventoryType and i found a problem. In the InventoryOpener interface, the default method fill (that is called by SpecialInventoryOpener) uses this code:

    default void fill(Inventory handle, InventoryContents contents) {        ClickableItem[][] items = contents.all();
        for(int row = 0; row &lt; items.length; row++) {            for(int column = 0; column &lt; items[row].length; column++) {                if(items[row][column] != null)                    handle.setItem(9 * row + column, items[row][column].getItem());            }        }    }

But this code can be used only for inventories with 9 columns, so inventorytypes like dropper aren't supported by this fill method, if you try to add an item in a dropper it gives error because of the plugin treats the inventory as if it has 9 columns (IndexOutOfBoundsException)

**Comment by MinusKube** [comment]
Indeed, it is a problem. I think you can currently workaround this by giving a size of 1 row and 9 columns to the inventory, but this is not very intuitive.

**Comment by ImDxni** [comment]
I directly rewrited the fill method
